features/gezag-minderjarige/gezag-niet-te-bepalen.feature

```diff
@@ -0,0 +1,111 @@
+#language: nl
+
+Functionaliteit: Als gebruiker wil ik alleen een gezaguitspraak krijgen wanneer die met zekerheid te bepalen is
+
+  Gezag is alleen met zekerheid te bepalen wanneer het gezag van de minderjarige onder Nederlands recht wordt bepaald.
+
+  Wanneer een minderjarige verhuist naar een ander land, neemt deze het gezag mee naar het nieuwe land.
+  Daarom is het gezag niet te bepalen van een minderjarige die uit een ander land is geimmigreerd.
+  Bij de Nederlandse overheid is niet bekend welk gezag de minderjarige uit het buitenland heeft meegenomen.
+
+
+    Achtergrond:
+      Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Aart' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En 'Gerda' en 'Aart' zijn met elkaar gehuwd
+
+
+  Regel: Gezag is niet te bepalen als de minderjarige in het RNI staat ingeschreven
+
+    Scenario: Minderjarige verblijft in het buitenland
+      Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is ingeschreven in de RNI
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                            |
+      | type        | GezagNietTeBepalen                                                                |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet of tijdelijk in Nederland woont. |
+
+    Scenario: Minderjarige verblijft tijdelijk in Nederland
+      Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is ingeschreven in de RNI
+      # En heeft een tijdelijk verblijfsadres in Nederland
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                            |
+      | type        | GezagNietTeBepalen                                                                |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet of tijdelijk in Nederland woont. |
+
+    Scenario: Verblijfplaats van minderjarige is onbekend
+      Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is ingeschreven in de RNI
+      # heeft een volledig onbekende verblijfplaats
```

Comment:
wat is de reden dat deze stap is gemarkeerd als commentaar?

Reply:
Dit is een illustratie van verschillende soorten inschrijvingen in de RNI. We hebben het nu toegevoegd aan de RNI inschrijving.